### PR TITLE
typedoc setup

### PR DESCRIPTION
**Pull request by SanchayPahalwani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,222 +53,13 @@ cruxClient.getCruxIDState().then((cruxIDState) => {
 })
 ```
 
+Find references to all available methods of [CruxClient](classes/cruxclient.html).
```
Comment (quixote911) [inline README.md]: Does this link to the docs?

File: src/index.ts [hunk 2]
```diff
@@ -281,6 +281,30 @@ export class CruxClient {
         }
     }
 
+    /**
```
Comment (nandubatchu) [inline src/index.ts]: validate if this is turning up correctly inside the docs.

**Comment by nandubatchu** [comment]
@SanchayPahalwani Please ensure that the required secrets are setup before this is merged.

**Comment by SanchayPahalwani** [comment]
> @SanchayPahalwani Please ensure that the required secrets are setup before this is merged.
Done

**Pull request merged**